source/fe/fe_nedelec.cc

```diff
@@ -3508,8 +3508,8 @@ FE_Nedelec<dim>::convert_generalized_support_point_values_to_dof_values(
                     system_matrix(i, j) +=
                       boundary_weights(q_point, j) *
                       lobatto_polynomials_grad[i + 1].value(
-                        this->generalized_face_support_points[face_no][q_point](
-                          1));
+                        this->generalized_face_support_points[face_no][q_point]
+                                                             [1]);
```

Comment:
Out of all of the places in this patch, this is the only one that incurs any whitespace changes.